CHANGELOG.md

```diff
@@ -1,24 +1,30 @@
+## 2020-12-10
+
+- New layout for metrics filtering tools! Instead of an overlapping sidebar that you can show / hide, we've embedded the controls for filtering by time range, clients, and operations directly into the page header. Now you can simply adjust the knobs without having to context-switch out of your current view.
+  ![Updated Filter UX](https://user-images.githubusercontent.com/1831709/101563762-f7282d00-3a1d-11eb-9a89-012280710c52.gif)
+
 ## 2020-11-16
 
-| New in the Explorer | |
-| :-- | :-- |
-| Don’t send unused variables over the wire | ![Screen Recording 2020-11-16 at 04 00 42 PM](https://user-images.githubusercontent.com/5922187/99323038-246e2a80-2826-11eb-9e86-01e8c6cd9b47.gif) |
-| Visual Query Plans (rendered using mermaid) | ![image](https://user-images.githubusercontent.com/5922187/99334771-b893be00-2835-11eb-89ef-593a061060ee.png) |
-| Table Mode column sorting | ![Screen Recording 2020-11-16 at 06 03 23 PM](https://user-images.githubusercontent.com/5922187/99336192-20e29f80-2836-11eb-8e2a-7c650834bc64.gif) |
-| Download arrays from Table Mode to CSVs | ![image](https://user-images.githubusercontent.com/5922187/99335997-12948380-2836-11eb-8a34-8ae4c8ab2442.png) |
-| Choose the percentile (p50, p90, p99, etc) that you want your field latency hints displayed from | ![image](https://user-images.githubusercontent.com/5922187/99337060-6ef7a300-2836-11eb-81eb-32f0ecfdf7e3.png) |
+| New in the Explorer                                                                              |                                                                                                                                                    |
```

Comment:
Apparently my markdown formatter in VSCode is not pleased with the previous formatting...  looks like rendering is unchanged.